protocol/x/listing/keeper/msg_create_market_permissionless_test.go

```diff
@@ -19,35 +24,74 @@ func TestMsgCreateMarketPermissionless(t *testing.T) {
 	tests := map[string]struct {
```

Comment:
in this test, maybe also check for market, clob pair, perpetual, and vault?

Reply:
Will be adding more integration tests in a follow up PR